site/docs/src/json/tenants/response.json

```diff
@@ -4,15 +4,26 @@
     "data": {
       "description": "No more secrets, Marty."
     },
+    "captchaConfiguration": {
+      "captchaMethod": "GoogleRecaptchaV3",
+      "enabled": true,
+      "secretKey": "6LdYWpMbAAAAAKhcksgk70us00012r66r96tt7rp",
```

Comment:
Are these keys live? You might want to change them to `YOURKEY` or something similar.

Reply:
And rotate your keys.

Reply:
They totally fake.  I make them up using IJ's random CUID or UUID depending.